app/models/event.rb

```diff
@@ -68,13 +85,14 @@ def total_github_stats
     stats
   end
 
-  def github_stats
-    @github_stats ||= fetch_github_stats
+  def engagement_stats
+    @engagement_stats ||= fetch_engagement_stats
   end
 
-  def fetch_github_stats
+  def fetch_engagement_stats
     stats = {}
 
+    stats[:favorites] = featured_projects_favorites_stats
```

Comment:
This naming is a bit ambiguous -- perhaps `stats[:featured_favorites]` for `featured_projects_favorites_stats` and `stats[:platform_favorites]` or `stats[:all_favorites]` for the all-inclusive method proposed above?